documentation/docs/steps/detectExecuteScan.md

```diff
@@ -1,6 +1,11 @@
-# ${docGenStepName}
+# detectExecuteScan
```

Comment:
The docs are generated when deployed to GH pages, do not commit generated docs.

Reply:
Thanks for the heads up. I will remove this from the commit